pkg/controller/che/gateway_permission.go

```diff
@@ -0,0 +1,67 @@
+package che
+
+import (
+	orgv1 "github.com/eclipse-che/che-operator/pkg/apis/org/v1"
+	"github.com/eclipse-che/che-operator/pkg/deploy"
+	"github.com/eclipse-che/che-operator/pkg/deploy/gateway"
+	"github.com/eclipse-che/che-operator/pkg/util"
+	rbac "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func (r *ReconcileChe) reconcileGatewayPermissions(deployContext *deploy.DeployContext) (bool, error) {
```

Comment:
pls. don't use `SyncClusterRoleBindingAndAddFinalizerToCluster` but something similar to https://github.com/eclipse-che/che-operator/blob/main/pkg/controller/che/workspace_namespace_permission.go#L77-L112

Reply:
fixed